### PR TITLE
Spelling comments c

### PR DESCRIPTION
**Pull request by jsoref**
This PR should be purely comment changes for words (or families of words) that start w/ `c`

To make my life easier, these initial comment focused PRs will have 2 commits:
* one which is the fixes
* and one which is the copyright bumping

For more information, see #5722 

For reference, these [commits](https://github.com/jsoref/openj9/commits/spelling-full) are generated using https://github.com/jsoref/spelling `f` / `fchurn`. That branch will grow as I make progress (I've finished `s`).

### CODE REVIEW

File: runtime/compiler/z/runtime/PicBuilder.m4 [hunk 1]
```diff
@@ -1193,7 +1193,7 @@ ifdef([OMR_GC_COMPRESSED_POINTERS],[dnl
 
     MTComputeStaticAddress(AddressLoad,Obj)
 ifdef([OMR_GC_COMPRESSED_POINTERS],[dnl
-    LLGF    r3,0(,r2)                        # load comprssed val
+    LLGF    r3,0(,r2)                        # load compressed val
     SLLG    r3,r3,0(r1)                       # decompress
```
Comment (jsoref) [inline runtime/compiler/z/runtime/PicBuilder.m4]: This whitespace isn't my fault (I'm pretty sure I've broken some others)

File: runtime/compiler/z/codegen/J9BCDTreeEvaluator.cpp [hunk 2]
```diff
@@ -2387,7 +2387,7 @@ J9::Z::TreeEvaluator::isZonedOperationAnEffectiveNop(TR::Node * node, int32_t sh
  *
  * \param node                  the BCDCHK node
  * \param cg                    codegen object
- * \param numCallChildre        number of callNode children
+ * \param numCallChildren        number of callNode children
```
Comment (jsoref) [inline runtime/compiler/z/codegen/J9BCDTreeEvaluator.cpp]: This is the only indentation failing I know of

File: debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9MemTagCommands.java [hunk 3]
```diff
@@ -802,7 +802,7 @@ public long getAllocSize () {
 	}
 
 	/**
-	 * CustomComparetor is used to sort J9MemTags depending on their allocated sizes. 
+	 * CustomComparator is used to sort J9MemTags depending on their allocated sizes.
```
Comment (pshipton) [inline debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9MemTagCommands.java]: I think this, and the next, should be two words: Custom Comparator
Comparator should be capitalized as it refers to the interface used below.

File: runtime/compiler/z/runtime/PicBuilder.m4 (hunk 1 above)
Comment (fjeremic) [reply runtime/compiler/z/runtime/PicBuilder.m4]: Should be fine. Thanks for your contribution.

File: runtime/gc_base/StringTable.cpp [hunk 4]
```diff
@@ -981,7 +981,7 @@ j9gc_allocStringWithSharedCharData(J9VMThread *vmThread, U_8 *data, UDATA length
  * Determine the unicode length of a UTF8 string, while testing its compressability
  * @param data A pointer to a UTF8 string
  * @param length The length of the UTF8 string
- * @param isCompressable[out] Is the string compressable, or NULL
+ * @param isCompressible[out] Is the string compressible, or NULL
```
[the remark this answers is not in the document]
Comment (jsoref) [reply runtime/gc_base/StringTable.cpp]: I'll drop this for now.

File: runtime/compiler/z/codegen/J9BCDTreeEvaluator.cpp [hunk 5]
```diff
@@ -2387,7 +2387,7 @@ J9::Z::TreeEvaluator::isZonedOperationAnEffectiveNop(TR::Node * node, int32_t sh
  *
  * \param node                  the BCDCHK node
  * \param cg                    codegen object
- * \param numCallChildre        number of callNode children
+ * \param numCallChildren       number of callNode children
```
[the remark this answers is not in the document]
Comment (jsoref) [reply runtime/compiler/z/codegen/J9BCDTreeEvaluator.cpp]: Sigh, in that case, it should go to the `p` list. We'll see it later.

**Comment by jsoref** [comment]
@pshipton: Thanks for the review. I've addressed those items:
* moved to `p`
* waiting for code pass
* added spaces

**Comment by jsoref** [comment]
@pshipton while eclipse/openj9's travis is happy w/ this, my travis is not: https://travis-ci.org/jsoref/openj9/jobs/533127901#L5122
> error: module not found: jdk.hotspot.agent

-- It seems like you've had reports of that elsewhere as well, any idea what it means? (I haven't done any configuration of travis, it should have the same stuff as the eclipse travis) -- and it was happy in https://travis-ci.org/jsoref/openj9/builds/533076059

**Comment by pshipton** [comment]
@dnakamura might know what the travis problem is

**Comment by pshipton** [comment]
jenkins compile zlinux jdk8

**Pull request merged**